src/Patch/Cli/Command/ApplyCommand.php

```diff
@@ -114,10 +114,26 @@ protected function configure()
      */
     public function setPhpVersion($version)
     {
+        // Check if the current version has metadata in it
+        // example : 5.6.0-1ubuntu3.25
+        if (count(explode('-', $version)) > 1) {
+            $versionWithMetadata = explode('-', $version);
```

Comment:
Or another alternative is using strpos/substr.

```
$metadataPos = strpos($version, '-');
if ($metadataPos !== false) {
    $version = substr($version, 0, $metadataPos);
}
```

Using explode is nice and simple though :+1: